datacenter/ucp/2.1/guides/architecture.md

```diff
@@ -25,8 +25,11 @@ and Docker Compose.
 Docker UCP leverages the clustering and orchestration functionality provided
 by Docker.
 
+<a name="figure-architecture-2">
```

Comment:
You're changing from markdown to html, but what does this html version do that the markdown version doesn't already?
This takes more space and `name` attribute is deprecated in html5, so if this does exactly the same thing as the markdown version, you should revert this.